conta/use_cases/criar_conta_use_case.py

```diff
@@ -0,0 +1,36 @@
+
+from conta.exceptions.cpf_invalido_por_numero_de_caracteres import NumeroDeCaracteresCpf
+from conta.exceptions.cpf_invalido_por_tipo_de_caracter import CpfInvalidoTipoCaracter
+from conta.exceptions.cpf_nao_informado import CpfNaoInformado
+from conta.models import ContaCorrente
+from conta.use_cases.gerar_num_conta_use_case import GerarNumContaUseCase
+
+
+class CriarContaUseCase:
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.gerar_num_conta_use_case = GerarNumContaUseCase()
+
+    def execute(self, nome: str, cpf: str) -> ContaCorrente:
+        if len(cpf) != 11:
```

Comment:
Aqui a mesma coisa la d ecima
```suggestion
        if len(cpf) != 11:
```